mne/decoding/time_gen.py

```diff
@@ -256,7 +267,7 @@ def f(x):
         self.estimators_ = sum(out, list())
         return self
 
-    def predict(self, epochs, test_times=None, picks=None):
+    def predict(self, epochs, test_times=None, y=None, picks=None):
```

Comment:
It needs it if you haven't fitted first : 

```
gat=GeneralizationAcrossTime()
gat.predict(epochs, y=y)
```
will fit with y and then predict. See my last comments.

I can remove this last 'ehn' if you think the shortcut isn't necessary and could lead to confusion.